spark-etl/src/main/scala/geotrellis/spark/etl/hadoop/NetCdfHadoopInput.scala

```diff
@@ -1,14 +1,13 @@
 package geotrellis.spark.etl.hadoop
 
-import geotrellis.spark.io.hadoop.formats.NetCdfBand
 import geotrellis.spark.io.hadoop._
 import geotrellis.spark._
 import org.apache.hadoop.fs.Path
 import org.apache.spark.SparkContext
 import org.apache.spark.rdd.RDD
 
-class NetCdfHadoopInput extends HadoopInput[NetCdfBand, SpaceTimeKey] {
+class NetCdfHadoopInput extends HadoopInput[SpaceTimeInputKey, SpaceTimeKey] {
   val format = "netcdf"
 
-  def source(props: Parameters)(implicit sc: SparkContext): RDD[(NetCdfBand, V)] = sc.netCdfRDD(new Path(props("path")))
+  def source(props: Parameters)(implicit sc: SparkContext): RDD[(SpaceTimeInputKey, V)] = sc.netCdfRDD(new Path(props("path")))
```

Comment:
"rename SpaceTimeInputKey -> TemporalProjectedExtent"
Looks like the file got renamed but not the actual class